exec/table_exec_command.go

```diff
@@ -51,30 +54,34 @@ func listExecCommand(ctx context.Context, d *plugin.QueryData, h *plugin.Hydrate
 	var cmd *remote.Cmd
 
 	outR, outW := io.Pipe()
-	errR, errW := io.Pipe()
+	// errR, errW := io.Pipe()
 	defer outW.Close()
-	defer errW.Close()
+	// defer errW.Close()
 
 	var wg sync.WaitGroup
 
-	wg.Add(1)
-	go func() {
-		defer wg.Done()
-		err = outputIntoRow(ctx, d, outR, false)
-		if err != nil {
-			plugin.Logger(ctx).Error("listExecCommand", "outputLinesIntoRows", err)
-		}
-	}()
+	stdout := ""
+	// stdout, stderr := "", ""
 
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		err = outputIntoRow(ctx, d, errR, true)
+		buf := new(strings.Builder)
+		_, err := io.Copy(buf, outR)
 		if err != nil {
-			plugin.Logger(ctx).Error("listExecCommand", "outputLinesIntoRows", err)
+			plugin.Logger(ctx).Error("listExecCommand", "error reading output", err)
 		}
+		stdout = buf.String()
 	}()
 
+	// wg.Add(1)
```

Comment:
Please remove any unused code (if not useful)

Reply:
done